Makefile

```diff
@@ -167,6 +173,14 @@ ssh-mariadb::
 ssh-webserver::
 	docker-compose exec -w /etc/nginx webserver sh
 
+###############################################################################
+#                                CLONE                                        #
+###############################################################################
+clone::
+	@docker-compose exec --user $(HOST_USER) php-fpm ssh-agent /bin/bash -c "\
+		./flow clone:list; \
+		./flow clone:preset --yes"
```

Comment:
It would be great to be able to pass a preset via `make clone preset=dev`.

Reply:
I don't get it running. Maybe we should take a look together.

Reply:
done. :)